migrator/clone/postgres/db_clone_manager_impl.go

```diff
@@ -162,24 +162,26 @@ func (d *dbCloneManagerImpl) GetCloneToMigrate(rocksVersion *migrations.Migratio
 		// Rocks more recently updated than Postgres so need to migrate from there.  Otherwise, Postgres is more recent
 		// so just fall through to the rest of the processing.
 		if currClone.GetMigVersion().LastPersisted.Before(rocksVersion.LastPersisted) {
-			// Rollback is not enabled, so process the Rocks -> Postgres migration on CurrentClone.
-			if !d.rollbackEnabled() {
-				return CurrentClone, true, nil
-			}
-
-			// Create a temp clone for processing of current
-			// Seed it from an empty database because we need to run migrations from Rocks to Postgres.
-			if !d.databaseExists(TempClone) {
-				err := pgadmin.CreateDB(d.sourceMap, d.adminConfig, pgadmin.EmptyDB, TempClone)
+			// We want to start fresh as we are migrating from Rocks->Postgres.  So any data that exists in
```

Comment:
These are the changes from https://github.com/stackrox/stackrox/pull/3314.  They are here for testing since that one hasn't been merged yet.